float64[] times          # N timestamps (in seconds)
float32[] orientations   # Nx3 orientation matrix
